apps/site/src/app/(main)/mentor/Form/ProfileInformation.tsx

```diff
@@ -0,0 +1,41 @@
+import TextInput from "@/lib/components/forms/TextInput";
+import styles from "@/lib/components/forms/shared/Form.module.scss";
+
+export default function ProfileInformation() {
+	return (
+		<div className="flex flex-col gap-5 w-11/12">
+			<p className="text-4xl m-0 font-bold max-[700px]:text-3xl">Links</p>
+
+			<TextInput
+				name="linkedin"
+				labelClass={styles.label}
+				labelText="LinkedIn"
+				inputClass={styles.input}
+				containerClass="flex flex-col w-full"
+				isRequired={false}
+				type="url"
+				placeholder="https://"
+			/>
+			<TextInput
+				name="github"
+				labelClass={styles.label}
+				labelText="Github"
```

Comment:
Chore: please stylize as "GitHub", maybe also include the word profile (same for LinkedIn).
Question: will the hacker application also ask for GitHub profile and portfolio separately?

Reply:
There is one line for Github/Portfolio in the  hacker application.